tools/docker/tests/test_flows.sh

```diff
@@ -79,6 +79,11 @@ test_ap_config_renew() {
     # Wait a bit for the renew to complete
     sleep 3
 
+    TMP=$(send_CAPI_command repeater1 "dev_get_parameter,program,map,ruid,0x000000000000,ssid,Multi-AP-24G-1,parameter,macaddr")
```

Comment:
(nitpick) it would make more sense to do this check after the checks of the log file.